main.tf

```diff
@@ -1,5 +1,7 @@
 provider "aws" {
   region = var.region
+  access_key = var.AWS_ACCESS_KEY_ID
+  secret_key = var.AWS_SECRET_ACCESS_KEY
```

Comment:
So I think this was only necessary because I created Terraform variables vs. environmental ones in in my cloud variable set...derp.